.github/workflows/ci.yaml

```diff
@@ -108,3 +118,62 @@ jobs:
                     DATABASE_URL: "pgsql://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/testdb?server_version=10"
                     # Required by old repository tests
                     DATABASE: "pgsql://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/testdb"
+
+    integration-tests-mysql:
+        name: MySQL integration tests
+        needs: tests
+        services:
+            mysql:
+                image: mysql:5.7
```

Comment:
A follow up draft PR with `mysql:8.0` is necessary to show that there are tests that will fail on MySQL 8.